conf/biowulf.config

```diff
@@ -0,0 +1,73 @@
+
+params {
+  config_profile_description = 'Biowulf nf-core config'
+  config_profile_contact = 'staff@hpc.nih.gov'
+  config_profile_url = 'https://hpc.nih.gov/apps/nextflow.html'
+  max_memory = '224 GB'
+  max_cpus = 32
+  max_time = '72 h'
+
+  igenomes_base = '/fdb/igenomes/'
+}
+
+// use a local executor for short jobs. For this the
+// settings below may have to be adapted to the allocation for
+// the main nextflow job
+executor {
+    $local {
+        queueSize = 100
+        memory = '36 G'
+        cpus = '24'
+
+    }
+    $slurm {
+        queue = 'norm'
+        queueSize = 200
+        pollInterval = '2 min'
+        queueStatInterval = '5 min'
+        submitRateLimit = '6/1min'
+        retry.maxAttempts = 1
+    }
+}
+
+singularity {
+    enabled = true
+    autoMounts = true
+    cacheDir = "/data/$USER/singularity"
+    envWhitelist='https_proxy,http_proxy,ftp_proxy,DISPLAY,SLURM_JOBID,SINGULARITY_BINDPATH'
+}
+
+env {
+    SINGULARITY_CACHEDIR="/data/$USER/singularity"
+    PYTHONNOUSERSITE = 1
+}
+
+profiles {
+    biowulflocal {
+        process.executor = 'local'
+        process.cache = 'lenient'
+
+    }
+
+    biowulf {
+        process {
+            executor = 'slurm'
+            errorStrategy = { task.exitStatus in 137..140 ? 'retry' : 'finish' }
```

Comment:
This should be left to the [pipeline defaults](https://github.com/nf-core/rnaseq/blob/3bec2331cac2b5ff88a1dc71a21fab6529b57a0f/conf/base.config#L17), unless you explicitly want to exclude 130-136?

Reply:
Removed, thanks for suggestion.